.buildkite/dev-docker/pipeline.yml

```diff
@@ -1,6 +1,8 @@
 steps:
   - input: "Build parameters"
-    if: build.source != "schedule"
+    # If the build is triggered by merging a PR to master, we want to build from the master branch.
+    # If the build is triggered by a schedule, we want to build from the current branch
+    if: '!((build.source == "webhook" && build.branch == "master") || build.source == "schedule")'
```

Comment:
Nit: The second comment is misleading to me as the current branch in case of scheduled build is... master. To me the point is we don't want to ask for inputs (and thus stop the pipeline) in case of scheduled builds and builds triggered by a PR merge to master.